benchmarks/nel/project.yml

```diff
@@ -48,10 +55,13 @@ commands:
     script:
       - "unzip -q -d assets/reddit assets/reddit.zip"
```

Comment:
Please add `-o` to always overwrite or something else so repeated runs don't ask for input.